org/allPRs.ts

```diff
@@ -49,8 +49,9 @@ export const rfc7 = async () => {
 // https://github.com/artsy/peril-settings/issues/13
 export const rfc13 = async () => {
   const pr = danger.github.pr
+  const isRenovate = pr.user.login.toLowerCase().includes("renovate")
   const wipPR = pr.title.includes("WIP ") || pr.title.includes("[WIP]")
-  if (!wipPR && pr.assignee === null) {
+  if (!isRenovate && !wipPR && pr.assignee === null) {
```

Comment:
this could check if the user is a bot instead, then it can do it for any automated PRs?